pilot/pkg/model/sidecar_test.go

```diff
@@ -2517,7 +2538,7 @@ func benchmarkConvertIstioListenerToWrapper(b *testing.B, vsNum int, hostNum int
 		})
 	}
 	ps := NewPushContext()
-	ps.virtualServiceIndex.publicByGateway["default"] = cfgs
```

Comment:
it is a bug, see here

https://github.com/istio/istio/blob/897ae22b5019ead245ae26553dd4dd5ec45c50f6/pilot/pkg/model/virtualservice.go#L82